### DISCUSSION TITLE
rec_model 换成 ch_PP-OCRv4_rec_server_infer 之后 卡住不动了，GPU true也非常慢，只有6k的图片才能结束，16k的图片都会卡住 ch_PP-OCRv4_rec_infer 小模型没这个问题，ch_PP-OCRv4_det_infer 也没这个问题

### DISCUSSION THREAD

**Discussion by alexhmyang** [PaddlePaddle/PaddleOCR#13558] [Q&A]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

![image](https://github.com/user-attachments/assets/88faab9d-9965-4859-b8b0-abf010fafa41)
python tools/infer/predict_system.py --use_gpu=True --image_dir="./doc/imgs/11.jpg" --det_model_dir="./ch_PP-OCRv4_det_server_infer/" --cls_model_dir="./ch_ppocr_mobile_v2.0_cls_infer/" --rec_model_dir="./ch_PP-OCRv4_rec_server_infer/" --use_angle_cls=true

rec_model 换成 ch_PP-OCRv4_rec_server_infer 之后 卡住不动了，GPU true也非常慢，只有6k的图片才能结束，16k的图片都会卡住 ch_PP-OCRv4_rec_infer 小模型没这个问题，ch_PP-OCRv4_det_infer 也没这个问题 

![image](https://github.com/user-attachments/assets/4bea080d-76d7-4862-b792-db6c0f91e928)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SWHL** [comment]
猜测是机器内存和GPU应该打满了，请尝试换一个小尺寸的图像，推理一下。
如果不会卡住，则原因是图像太大，机器在推理，耗时较长。
如果仍会卡住，则才可能是环境问题。

**Reply by alexhmyang** [reply]
我就是用你们 doc/img里的 小图测的，不到 50k, 非常慢，你们自己试一下再跟我说

![11](https://github.com/user-attachments/assets/44314966-d7ab-4c1d-8d66-8e127067bf2d)

**Reply by SWHL** [reply]
我们这挺快的

**Reply by alexhmyang** [reply]
什么配置， 我这 4070Ti 12g都不行

**Reply by SWHL** [reply]
你用百度的AI Studio试试吧，这样大家好对齐结果，排查问题

**Comment by yuqangy123** [comment]
我将inference模型转换成ONNX格式模型之后，GPU推理速度变成了秒级别。。。

**Reply by alexhmyang** [reply]
快了还是慢了？啥意思？
